tensorflow/core/graph/graph_constructor.cc

```diff
@@ -86,7 +86,8 @@ class GraphConstructor {
           return_nodes(in.return_nodes),
           importing(true),
           validate_colocation_constraints(in.validate_colocation_constraints),
-          validate_shape(in.validate_shape) {}
+          validate_shape(in.validate_shape),
+	  bind_device(in.bind_device) {}
```

Comment:
Align with the previous line (and also, call this `default_device` as per comment in `c_api.h`)